website/source/docs/providers/aws/r/db_instance.html.markdown

```diff
@@ -45,7 +45,7 @@ The following arguments are supported:
 * `name` - (Optional) The DB name to create. If omitted, no database is created
     initially.
 * `password` - (Required) Password for the master DB user. Note that this may
-    show up in logs, and it will be stored in the state file.
+    show up in logs, and it will be stored in the state file. Maximum 41 characters.
```

Comment:
Keep in mind this is a resource covering all engines, therefore this statement is not always true according to the official docs since not everyone may use MySQL:
http://docs.aws.amazon.com/AmazonRDS/latest/CommandLineReference/CLIReference-cmd-ModifyDBInstance.html

**MySQL**
Constraints: Must contain from **8 to 41** alphanumeric characters.

**Oracle**
Constraints: Must contain from **8 to 30** alphanumeric characters.

**SQL Server**
Constraints: Must contain from **8 to 128** alphanumeric characters.

**PostgreSQL**
Constraints: Must contain from **8 to 128** alphanumeric characters.